src/main/java/org/broadinstitute/hellbender/engine/datasources/ReferenceMultiSource.java

```diff
@@ -53,6 +54,16 @@ public ReferenceMultiSource( final PipelineOptions pipelineOptions, final String
         this.referenceWindowFunction = referenceWindowFunction;
     }
 
+    /**
+     * @param auth authentication information
+     * @param referenceURL the name of the reference (if using the Google Genomics API), or a path to the reference file
+     * @param referenceWindowFunction the custom reference window function used to map reads to desired reference bases
+     */
+    public ReferenceMultiSource(final AuthHolder auth, final String referenceURL,
+                                   final SerializableFunction<GATKRead, SimpleInterval> referenceWindowFunction) {
+        this(auth.asPipelineOptionsOldStyle(), referenceURL, referenceWindowFunction);
```

Comment:
but this doesn't convey the fact that it's not actually as pipeline options! It's "as pipelineoptions if there's an API key, or NULL If you have a client-secrets.json (or neither)". 

Renaming to `asPipelineOptionsDeprecated` to avoid time-relatedness while still strongly suggesting that you really should not be using this method in any newly written code (though I'll take other naming suggestions if you have some). As the comments in that method explain, the only reason I didn't mark it @deprecated is that otherwise our code won't compile (yay -Werr).